channel/channel.go

```diff
@@ -115,11 +115,6 @@ func (c Channel) MyDestination() types.Destination {
 	return types.AddressToDestination(c.Participants[c.MyIndex])
 }
 
-// TheirDestination returns the destination of the ledger counterparty
-func (lc TwoPartyLedger) TheirDestination() types.Destination {
-	return types.AddressToDestination(lc.Participants[(lc.MyIndex+1)%2])
-}
-
```

Comment:
Was this dead code?

Reply:
Never mind, I see that it is a method on the type you are removing 🤦

Reply:
I think so. I believe it was added preemptively as part of a designed API for the TPL. (seems like one might want to know __their__ address).